buildCommit.js

```diff
@@ -38,7 +48,7 @@ module.exports = function buildCommit(answers, config) {
   }
 
   // Hard limit this line
-  var head = (answers.type + addScope(answers.scope) + addSubject(answers.subject)).slice(0, maxLineWidth);
+  var head = (answers.type + addScope(answers.scope) + addTicketNumber(answers.ticketNumber, config) + addSubject(answers.subject)).slice(0, maxLineWidth);
```

Comment:
We should also discuss the location of the label `TICKET`. It may be important for you to have in the first line. 
IMO it should be fine to have in there in cases where config allows ticket number. In  all other cases, users can specify issues closed on the footer. This is another way. I am fine about this Simon but just get a second opinion from any other maintainer interested in reviewing this PR ok? 
cc @uglow

Reply:
This would be solved by the issue #49 right?